### DISCUSSION TITLE
New text(area) UI element for Field Layouts

### DISCUSSION THREAD

**Discussion by bertoost** [craftcms/cms#12870] [Ideas]
It's great we can have some UI elements into field layouts, but it would be nice if we also can add a simple text to the UI between fields, to explain fields for example.

Would be nice this field's content to could contain markdown or maybe it could render a Redactor field when installed?

**Comment by outline4** [comment]
+1  

it should display the same as the instructions for a field. 

I was very astonished that this does not exist yet.
